src/contexts/SuccessRateContext.ts

```diff
@@ -30,35 +38,54 @@ export interface SuccessRates extends SuccessRatesByType {
 interface SuccessRateContextValue {
   values: SuccessRates;
   loading: boolean;
+  thresholds: ThresholdSettings;
   updateSuccessRate: (type: SuccessRateTypes, id: number, successRate: number | undefined) => void;
+  updateThresholds: () => void;
 }
 
 export const defaultValues: SuccessRates = {
   checks: {},
   probes: {},
   defaults: {
-    thresholdColor: config.theme2.colors.text.disabled,
-    value: 0,
-    displayValue: 'N/A',
+    reachabilityValue: 0,
+    reachabilityDisplayValue: 'N/A',
     noData: true,
-    icon: 'minus',
+    uptimeValue: 0,
+    uptimeDisplayValue: 'N/A',
   },
 };
 
-const updateSuccessRate = (type: SuccessRateTypes, id: number, successRate: number | undefined) => {
-  const thresholdColor = getSuccessRateThresholdColor(successRate);
+export const defaultThresholds: ThresholdSettings = {
+  uptime: {
+    upperLimit: 90,
```

Comment:
Are these the right defaults? I think the current defaults are 99.5, 99, and 75? Not sure. The latency defaults also seem a little small (if they are in `ms`)

Reply:
Good catch 👍